### ISSUE TITLE
Parameter -c changes all-xccdf.xml

### ISSUE THREAD

**Issue by MajoDurco** [upgrades-migrations/preupgrade-assistant#247]
Main problem is that if PA run with --contents parameter which takes path to all-xccdf.xml, then this file is changed even though it should not be. When you run PA for a second time it throws the traceback.
``` python
Traceback (most recent call last):
  File "/usr/bin/preupg", line 35, in <module>
    sys.exit(main())
  File "/usr/bin/preupg", line 21, in main
    ret = app.run()
  File "/usr/lib/python2.6/site-packages/preupg/application.py", line 748, in run
    retval = self.scan_system()
  File "/usr/lib/python2.6/site-packages/preupg/application.py", line 507, in scan_system
    self.run_scan_process()
  File "/usr/lib/python2.6/site-packages/preupg/application.py", line 260, in run_scan_process
    self.run_scan(function=self.scanning_progress.show_progress)
  File "/usr/lib/python2.6/site-packages/preupg/application.py", line 276, in run_scan
    return ProcessHelper.run_subprocess(cmd, print_output=False, function=function)
  File "/usr/lib/python2.6/site-packages/preupg/utils.py", line 374, in run_subprocess
    function(stdout_data.decode(settings.defenc))
  File "/usr/lib/python2.6/site-packages/preupg/scanning.py", line 101, in show_progress
    self.output_data.append(u'{0}:{1}'.format(self.names[xccdf_rule], stdout_data.strip()))
```

**Comment by AloisMahdal** [comment]
BTW, this was reported in august downstream as rhbz#1368823.

**Comment by bocekm** [comment]
I suspect this line https://github.com/upgrades-migrations/preupgrade-assistant/blob/master/preupg/application.py#L458

**Comment by AloisMahdal** [comment]
qa_ack: downstream suite needs to work around this, so removing it would be nice, and would actually constitute the verification test itself.

**Comment by MajoDurco** [comment]
Solved #281.
